Projects/My_website/myWebsite/blog/views.py

```diff
@@ -18,3 +19,11 @@ class crudBlogApi(viewsets.ModelViewSet):
     serializer_class = blogSerializer
     authentication_classes = [TokenAuthentication]
     permission_classes = [IsAuthenticated]
+
+    def get(self,request):
+        userid = request.user.id
+        datalist = list(self.queryset)
+        serializers = blogSerializer(datalist,many=True)
```

Comment:
@codal-uchoudhary what is this many=True used for? Why we use this?

Reply:
> @codal-uchoudhary what is this many=True used for? Why we use this?

@abhi-codal , we use (many=True ) to tell the serializer that queryset may contain more than one items , so serializer convert the data accordingly. if we don't use (Many=True) serializer return only single item.